### ISSUE TITLE
AmberLeap: need to adapt leaprc path for AmberTools 17

### ISSUE THREAD

**Issue by graik** [graik/biskit#21]
tested with AmberTools 17

```
test_AmberLeap_findLeaprc (__main__.Test)
AmberLeap.findLeaprc test ... 
target leaprc: /opt/amber16/dat/leap/cmd/oldff/leaprc.ff10
ok
test_AmberLeap_run (__main__.Test)
AmberLeap.run test ... ERROR

======================================================================
ERROR: test_AmberLeap_run (__main__.Test)
AmberLeap.run test
----------------------------------------------------------------------
Traceback (most recent call last):
  File "AmberLeap.py", line 227, in test_AmberLeap_run
    debug=self.DEBUG)
  File "AmberLeap.py", line 125, in __init__
    self.leaprc = self.findLeaprc( leaprc or self.exe.leaprc )
  File "AmberLeap.py", line 178, in findLeaprc
    'Check exe_tleap.dat or provide an explicit leaprc parameter!'
LeapError: Could not find Forcefield definition ff03.r1. Check exe_tleap.dat or provide an explicit leaprc parameter!

----------------------------------------------------------------------
Ran 2 tests in 0.004s

FAILED (errors=1)
```

**Comment by graik** [comment]
fixed in `biskit3` branch but not yet in master

**Comment by graik** [comment]
closing as this will not be back-ported to 2.x but is solved for upcoming 3.0 release
